src/pages/mastery/Plant.tsx

```diff
@@ -198,14 +203,13 @@ function Plant() {
     const res = await PLANT_SERVICES.updatePlantbyId(orgID, newPlant.plantId, body);
     if (res.statusCode === 200) {
       fetchPlantDataByOrgId(paginationData?.current_page);
-      setNewPlant(initialState);
+      handleClear();
       setEditPlant(false);
       setShowEditSuccessModal(true);
     }
   };
 
   const onCloseEditModal = () => {
-    setNewPlant(initialState);
```

Comment:
did you forgot to add `handleClear();` or why its removed `setNewPlant(initialState);`